what4/test/ExprBuilderSMTLib2.hs

```diff
@@ -1191,7 +1191,7 @@ testResolveSymBV searchStrat =
 main :: IO ()
 main = do
   testLevel <- TestLevel . fromMaybe "0" <$> lookupEnv "CI_TEST_LEVEL"
-  let solverNames = SolverName <$> [ "cvc4", "yices", "z3" ]
+  let solverNames = SolverName <$> [ "cvc4", "yices", "z3" ] -- TODO RGS: Add cvc5 here?
```

Comment:
Indeed, we might want to follow this TODO's advice and add `cvc5` to this list. I'm not exactly sure if it will work out of the box, but it would be worth a try!